dev-resources/template/1_db.yml

```diff
@@ -24,14 +24,28 @@ Parameters:
   DBEngineVersion:
     Description: PostgreSQL engine version on RDS
     Type: String
-    Default: "12.9"
+    Default: "14.3"
     AllowedValues:
       - "12.9"
       - "14.3"
   DBInstanceClass:
     Description: For provisioned aurora, the instance class to use
     Type: String
-    Default: "db.r4.large"
+    Default: "db.r5.large"
+  DBInstanceReplica:
+    Description: Whether or not to create a second DB instance for use as a read-replica
+    Type: String
+    AllowedValues:
+      - true
+      - false
+    Default: false
+  DBInstancePublic:
```

Comment:
I _would_ say to make the public align with the read replica only, or let them be set individually, but considering that you cant actually specify which instance is which i suppose its moot.

Reply:
Yep, this is exactly why the public applies to both, only Jeff knows which will be which.

Reply:
exactly